src/rules/quotemarkRule.ts

```diff
@@ -91,6 +93,15 @@ export class Rule extends Lint.Rules.AbstractRule {
         const quoteMark = getQuotemarkPreference(args);
         const jsxQuoteMark = getJSXQuotemarkPreference(args);
 
+        if (sourceFile.fileName.endsWith(".json")) {
+            return this.applyWithFunction(sourceFile, walk, {
+                avoidEscape: true,
+                avoidTemplate: true,
+                jsxQuoteMark: '"',
+                quoteMark: '"',
+            });
```

Comment:
Shouldn't TypeScript already give errors for invalid JSON? Can the rule skip these files altogether?

I'm also wondering if all rules should skip JSON files, but that's a question for another issue...

Reply:
Created a separate issue for all rules: https://github.com/palantir/tslint/issues/4426

Reply:
Thanks!